src/java/org/apache/cassandra/auth/AuthConfig.java

```diff
@@ -133,4 +124,21 @@ public static void applyAuth()
         cidrAuthorizer.validateConfiguration();
         DatabaseDescriptor.getInternodeAuthenticator().validateConfiguration();
     }
+
+    public static <T> T authInstantiate(ParameterizedClass authCls, Class<T> defaultCls) {
```

Comment:
can be private ?

Reply:
Yes. It could also be moved to `FBUtilities` as this is generic and works with any `ParameterizedClass`. I've changed the visibility but I'll postpone the refactor to `FBUtilities`.

Reply:
yeah, not a huge fan of polluting FBUtilities.  better to try keep things in package, even if it means duplicating small amounts of (very basic) code.